samples/qa-scroller/src/components/Controls/Controls.js

```diff
@@ -9,9 +9,10 @@ import ScrollModeSwitch from '../ScrollModeSwitch';
 
 const Controls = ({handleFocusableScrollbar, handleHeight, handleScrollMode, handleWidth, height, nativeScroll, width}) => {
 	const inputWidth = {width: '5em'};
+	const rowWidth = typeof window !== 'undefined' ? `${ri.scale(window.innerWidth)}px` : `${ri.unit(ri.scale(1920), 'rem')}`;
```

Comment:
We do have `ri.scaleToRem` instead of `ri.unit(ri.scale(1920), 'rem')`.

Reply:
On my PC, `window.innerWidth` value is 480 instead of 1920. So the header looks shrank. Does your sample look okay?

Reply:
Hello, I applied ri.scaleToRem.

Reply:
Regarding the window.innerWidth on our PC's it's 1920. I discussed a bit with the team regarding the Header for smaller widths and we think that it needs to be refactored. We shall do it in another user story. (as a background information, when this sample was created a few Sprints ago the same layout from Sandstone was used)

Reply:
Oh, never mind. I think my PC environment went something wrong. But yes, please go ahead to refactor this code in another ticket.